substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/pecoff/cv/CVSymbolSubsection.java

```diff
@@ -66,7 +66,7 @@ protected int computeContents(byte[] buffer, int initialPos) {
     public void logContents(DebugContext debugContext) {
         CVSectionImpl section = cvDebugInfo.getCVSymbolSection();
         for (CVSymbolSubrecord subcmd : subcmds) {
-            section.log(debugContext, "     [0x%08x]  %s", subcmd.getPos(), subcmd.toString());
+            section.log("     [0x%08x]  %s", subcmd.getPos(), subcmd.toString());
```

Comment:
I think you can also drop `debugContext` from the method signature.

Reply:
Done.